CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v5.0.0
+
+Changed
+- `Colopl\Spanner\Session` has been renamed to `Colopl\Spanner\SessionInfo`.
+- Default SessionPool was changed from `Google\Cloud\Spanner\Session\CacheSessionPool` to `Colopl\Spanner\CacheSessionPool`.
```

Comment:
How about adding a description of googleapis/google-cloud-php#5567 and googleapis/google-cloud-php#5595 as related issues here?

Reply:
I'll add https://github.com/googleapis/google-cloud-php/issues/5567.

https://github.com/googleapis/google-cloud-php/issues/5595 is not handled here so that will not be included.

Reply:
done